src/py/flwr/common/config.py

```diff
@@ -73,6 +76,42 @@ def get_project_config(project_dir: Union[str, Path]) -> Dict[str, Any]:
     return config
 
 
+def get_fused_config(run: Run, flwr_dir: Optional[Path]) -> Dict[str, str]:
+    """Merge the overrides from a `Run` with the config from a FAB.
+
+    Get the config using the fab_id and the fab_version, remove the nesting by adding
+    the nested keys as prefixes separated by dots, and fuse it with the override dict.
+    """
+    if not run.fab_id or not run.fab_version:
+        return {}
+
+    project_dir = get_project_dir(run.fab_id, run.fab_version, flwr_dir)
+
+    default_config = get_project_config(project_dir)["flower"]["config"]
```

Comment:
A project might not have `config` set, we should check for that.

Reply:
Oh absolutely, sorry for that, I'll correct it